package/MDAnalysis/analysis/density.py

```diff
@@ -507,17 +508,20 @@ def _set_user_grid(gridcenter, xdim, ydim, zdim, smin, smax):
            Now a staticmethod of :class:`DensityAnalysis`.
         """
```

Comment:
These updates to error checking need to be documented in the `Raises` section of the `DensityAnalysis` docstring, as they are now out of date.